hw/pci-bridge/simbridge.c

```diff
@@ -875,6 +875,31 @@ simbridge_poll_for_server(void *opaque)
               qemu_clock_get_ms(QEMU_CLOCK_REALTIME) + 1000);
 }
 
+/* advance model timers by 100us each 1s of vCPU time */
+#define TIMER_DELTA_US 100
```

Comment:
Even the scale factor is something the server side should handle.  Just give the model a 1s vCPU timer tick msg and let it do the scaling on the model side.

Reply:
I disagree - allowing to advance in different steps is more flexible

Reply:
perhaps I can abstract that out by giving vcpu time to this function